src/assets/components/_animatedLogo.scss

```diff
@@ -0,0 +1,34 @@
+.logoContainer {
```

Comment:
could you change this class to animatedLogo and nest all of the other classses beneath this? animatedLogo should be the only top level class. This will match with the rest of the scss

Reply:
See the latest commit.